cps/kobo.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import base64
+import copy
```

Comment:
My debugger tells me, this import isn't used, please check

Reply:
Done.